voluncheer/voluncheer/urls.py

```diff
@@ -42,4 +44,4 @@
     # path("chat/", include("chatroom.urls")),
     path("jobboard/", include("jobboard.urls")),
     path("map/", include("map.urls")),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Just curious, what is this line for?

Reply:
It allows you to access image in media folder via url when you are in static/local debugging environment.